src/lev-report/charts/LevUsage.js

```diff
@@ -24,15 +21,16 @@ const CustomLabel = ({ freq, ...props }) => (freq < 33) || (freq > 150) ?
   <g transform={`rotate(-45 ${props.x} ${1.1 * props.y})`}><VictoryLabel { ...props } /></g> :
   <VictoryLabel { ...props } />;
 
-const datatypeBar = data => data && data.length && <VictoryBar
-  data={data}
+const datatypeBar = ({ name, dailyUsage }) => dailyUsage && dailyUsage.length && <VictoryBar
+  key={name}
```

Comment:
Is `name` definitely unique?

Reply:
Yes, it's the dataset name: `[birth, death, marriage, partnership]`